components/automate-ui/src/app/modules/infra-proxy/infra-search-bar/infra-search-bar.component.scss

```diff
@@ -16,22 +16,21 @@
   position: relative;
 
   .search-button {
-    box-shadow: 0 1px 14px #E6EBEE;
     border: none;
     border-left: none;
     border-radius: 0px 4px 4px 0px;
     background: #E6EBEE;
     cursor: pointer;
-    width: 45px;
-    height: 42px;
+    width: 44px;
+    height: 43px;
     outline: none;
     position: absolute;
-    right: 1.5px;
-    top: 0.5px;
+    right: 0.8px;
+    top: 1px;
   }
 
   .chef-input {
-    border-radius: 4px 0px 0px 4px;
+    border-radius: 4px;
     width: 100%;
   }
 
```

Comment:
Lets revert all these changes from this file to master - this file, including the changes above, has been updated extensively in https://github.com/chef/automate/pull/4875

Reply:
Sure